### DISCUSSION TITLE
template validated successfully but no works

### DISCUSSION THREAD

**Discussion by hooray195** [projectdiscovery/nuclei#5520] [Q&A]
When i wate to use a template it said "syntax error".But i used  "-validate" flag, there was no errors.
![Snipaste_2024-08-14_17-29-15](https://github.com/user-attachments/assets/6c288c16-9c38-4729-8d1d-6dc68c0714b4)

**Comment by ehsandeep** [comment]
@hooray195 you can use `-v` flag to see the error details.

@dogancanbakir possible bug with validate option?

**Reply by dogancanbakir** [reply]
@ehsandeep Looks like.
@hooray195 Could you share the template? If you're not comfortable sharing on GH, please feel free to reach out over Discord.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
